lib/fluent/plugin/out_forward.rb

```diff
@@ -138,6 +139,10 @@ class ForwardOutput < Output
       config_param :weight, :integer, default: 60
     end
 
+    config_section :service_discovery do
```

Comment:
This define should be done via helper. See `parser` helper implementation.

Reply:
fixed https://github.com/fluent/fluentd/pull/2541/commits/54e8d537940b98c3d15c6d9d2f34e6c7790c0062